liquibase-core/src/main/java/liquibase/lockservice/StandardLockService.java

```diff
@@ -189,8 +216,12 @@ public boolean hasChangeLogLock() {
         return hasChangeLogLock;
     }
 
-    public boolean hasDatabaseChangeLogLockTable() throws DatabaseException {
-        if (hasDatabaseChangeLogLockTable == null) {
+    /**
+     * Check whether the databasechangeloglock table exists in the database.
+     * @param forceRecheck if true, do not use any cached information and check the actual database
+     */
+    public boolean hasDatabaseChangeLogLockTable(boolean forceRecheck) {
```

Comment:
I'm not sure this should be public. Whether we cache or not should maybe be an implementation-specific detail

Reply:
Certainly this can be private. I made it public because the method it overrides was also public. I'll make it private.